ccm_web/client/src/api.ts

```diff
@@ -55,18 +65,23 @@ export const getGlobals = async (key: string | undefined): Promise<Globals> => {
   return await resp.json()
 }
 
-const delay = async (ms: number): Promise<void> => {
-  await new Promise<void>(resolve => setTimeout(() => resolve(), ms))
+// usage:
+//   const data = await delay(nnnn).then(() => {return something})
+// const delay = async (ms: number): Promise<void> => {
+//   await new Promise<void>(resolve => setTimeout(() => resolve(), ms))
+// }
```

Comment:
```suggestion

```

Reply:
Are code comments considered poor practice now?  I feel like any time I leave commented code it's asked to be removed.

Reply:
Generally I won't leave code that is not used in the repo, But you can ask other developers what is the best practice here

Reply:
Don't they just get minified out in production anyways?

I might remember that once upon a time I left myself a comment somewhere (Narrator: he won't) and look it up in the history, but nobody else is going to know that once upon a time there was a comment that might have been helpful when they are looking at some code for the first time.

tldr; get off my lawn ;-)